app/models/concerns/wakeable/has_many.rb

```diff
@@ -8,6 +8,11 @@ module HasMany
       attr_accessor :has_many_label, :has_many_path
     end
 
+    def raw_aggregate_pageview_count
+      wakes_resources.reload
+      wakes_resources.sum("COALESCE((wakes_resources.document ->> 'pageview_count')::int, 0)")
```

Comment:
@benhutton I don't think we necessarily need to do anything in this PR, but if I understand this correctly, the COALESCE is essentially providing a default value to the `pageview_count` attribute. I wonder if it would be a good idea to establish a pattern for default values with jsonb attributes?

And, if we do this, would seeding the value upon creation be a better approach? And if so, how could we handle that?